ddtrace/contrib/anthropic/patch.py

```diff
@@ -0,0 +1,164 @@
+                    )
+                    span.set_tag_str(
+                        "anthropic.request.messages.%d.0.type" % (message_set_idx),
+                        "text",
+                    )
+                elif isinstance(message_set.get("content", None), list):
+                    for message_idx, message in enumerate(message_set.get("content", [])):
+                        if integration.is_pc_sampled_span(span):
+                            if message.get("type", None) == "text":
+                                span.set_tag_str(
+                                    "anthropic.request.messages.%d.%d.content" % (message_set_idx, message_idx),
+                                    integration.trunc(str(message.get("text", ""))),
+                                )
+                            elif message.get("type", None) == "image":
+                                span.set_tag_str(
+                                    "anthropic.request.messages.%d.%d.content" % (message_set_idx, message_idx),
+                                    "([IMAGE DETECTED])",
+                                )
+
+                        span.set_tag_str(
+                            "anthropic.request.messages.%d.%d.type" % (message_set_idx, message_idx),
+                            message.get("type", "text"),
+                        )
+        params_to_tag = {k: v for k, v in kwargs.items() if k != "messages"}
+        span.set_tag_str("anthropic.request.parameters", json.dumps(params_to_tag))
+
+        chat_completions = func(*args, **kwargs)
+        if isinstance(chat_completions, anthropic.Stream) or isinstance(
+            chat_completions, anthropic.lib.streaming._messages.MessageStreamManager
+        ):
+            return handle_stream_response(integration, chat_completions, args, kwargs, span)
+        else:
+            return handle_non_stream_response(integration, chat_completions, chat_messages, span)
+    except Exception:
+        span.set_exc_info(*sys.exc_info())
+        if integration.is_pc_sampled_llmobs(span):
+            integration.llmobs_set_tags(span, chat_messages, chat_completions, err=bool(span.error))
+        span.finish()
+        raise
+        # integration.metric(span, "dist", "request.duration", span.duration_ns)
+
+
+def handle_non_stream_response(integration, chat_completions, chat_messages, span):
+    try:
+        for idx, chat_completion in enumerate(chat_completions.content):
+            if integration.is_pc_sampled_span(span):
+                span.set_tag_str(
+                    "anthropic.response.completions.%d.content" % (idx),
+                    integration.trunc(getattr(chat_completion, "text", "")),
+                )
+            span.set_tag_str(
+                "anthropic.response.completions.%d.type" % (idx),
+                chat_completion.type,
+            )
+        usage = getattr(chat_completions, "usage", {})
+        if usage:
```

Comment:
To keep things consistent with the openai integration, can we move this code block to `AnthropicIntegration.record_usage(span, usage)`? Thanks!